src/Tribe/RSVP.php

```diff
@@ -534,31 +563,58 @@ public function generate_tickets( ) {
 
 
 			// After Adding the Values we Update the Transient
-			Tribe__Post_Transient::instance()->delete( $event_id, Tribe__Tickets__Tickets::ATTENDEES_CACHE );
+			Tribe__Post_Transient::instance()->delete( $post_id, Tribe__Tickets__Tickets::ATTENDEES_CACHE );
 		}
 
 		/**
 		 * Fires when an RSVP attendee tickets have been generated.
 		 *
-		 * @param int    $order_id ID of the RSVP order
-		 * @param int    $event_id ID of the event the order was placed for
+		 * @param int    $order_id              ID of the RSVP order
+		 * @param int    $post_id               ID of the post the order was placed for
 		 * @param string $attendee_order_status 'yes' if the user indicated they will attend
 		 */
-		do_action( 'event_tickets_rsvp_tickets_generated', $order_id, $event_id, $attendee_order_status );
+		do_action( 'event_tickets_rsvp_tickets_generated', $order_id, $post_id, $attendee_order_status );
+
+		// @todo: this positive stati should be filterable
+		$send_mail_stati = array('yes');
```

Comment:
`array( 'yes' )`

Also: It _looks_ like you have a filter for this down below. Is this comment still needed?